.github/workflows/test-zkvm-miden.yml

```
@@ -0,0 +1,20 @@
+name: Test and clippy Miden
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test:
+    uses: ./.github/workflows/test-miden.yml
+    permissions:
+      contents: read
+      packages: write
+    with:
+      zkvm: miden
+      toolchain: 1.88.0
+      test_ere_dockerized: false
+      default_features: false
```

We could set it to `true` becasue there is nothing block the CI to run with the default feature
```suggestion
      default_features: true
```